hardware/raspberrypi/booteeprom.md

```diff
@@ -44,17 +48,18 @@ sudo systemctl unmask rpi-eeprom-update
 
 The `FREEZE_VERSION` option in the EEPROM config file may be used to indicate that the EEPROM should not be updated on this board. 
 
+Note that by default, updating the bootloader (automatically or manually) will retain any custom configuration options of the previous installed version. You can override the migration by manually updating and using the `-d` option with `rpi-eeprom`. This will force the updated bootloader to use its inbuilt defaults.
+
 ## Write protection of EEPROM
 
 There is no software write protection for the boot EEPROM but there will be a mechanism in Raspberry Pi OS to skip any future updates to the EEPROM. However, it is possible to physically write-protect both EEPROMs via a simple resistor change on the board. Details will be published in the [schematics](./schematics/README.md).
```

Comment:
"there will be a mechanism in Raspberry Pi OS to skip any future updates to the EEPROM"

@timg236 Is the the "`FREEZE_VERSION` option in the EEPROM config file" that's described earlier in this page, or is this referring to something else?

Reply:
Alternatively, rid of this entire section. People who care about this sort of thing already know about EEPROMs and for less experienced users doing this or FREEZE_EEPROM will cause confusion.

Reply:
Revmoed